keys/thomas.velich.pub

```diff
@@ -0,0 +1 @@
+ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAILdFlopWNgDybXfgrwmJdpaDHAhe3qRyJys+is2KfUX2 thomas.velich@goodeggs.com
```

Comment:
[Interesting](https://medium.com/risan/upgrade-your-ssh-key-to-ed25519-c6e8d60d3c54)! No one else uses this. Seems reasonable though. 👍🏼

Reply:
seemed odd to me, but saw they recommended in the most recent gh docs if available so rolled with it